modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpiMBeanTest.java

```diff
@@ -81,4 +86,62 @@ public void testMBean() throws Exception {
             stopAllGrids();
         }
     }
+
+        /**
+      * Tests TcpDiscoverySpiMBean#excludeNode.
+      *
+      * @throws Exception if fails.
+      */
+        public void testNodeExclusion() throws Exception {
+            int srvCnt = 2;
+
+            IgniteEx grid0 = (IgniteEx)startGrids(srvCnt);
+
+            IgniteEx client = null;
+
+            try {
+                Ignition.setClientMode(true);
+
+                client = startGrid(srvCnt);
+            }
+            finally {
+                Ignition.setClientMode(false);
+            }
+
+            MBeanServer srv = ManagementFactory.getPlatformMBeanServer();
+
+            ObjectName spiName = U.makeMBeanName(grid0.context().igniteInstanceName(), "SPIs",
+                TcpDiscoverySpi.class.getSimpleName());
+
+            TcpDiscoverySpiMBean bean = JMX.newMBeanProxy(srv, spiName, TcpDiscoverySpiMBean.class);
+
+            assertEquals(grid0.cluster().forServers().nodes().size(), srvCnt);
+
+            assertEquals(grid0.cluster().forClients().nodes().size(), 1);
+
+            bean.excludeNode(client.localNode().id().toString());
+
+            assertTrue(GridTestUtils.waitForCondition(() ->
+                grid0.cluster().forClients().nodes().isEmpty(), 5_000));
+
+            assertTrue(strLog.toString().contains("Node failed through JMX interface"));
+
+            assertEquals(grid0.cluster().forClients().nodes().size(), 0);
+
+            bean.excludeNode(new UUID(0, 0).toString());
+
+            bean.excludeNode("fakeUUID");
+
+            assertEquals(grid0.cluster().forServers().nodes().size(), srvCnt);
+
+            ClusterNode node = grid0.cluster().forServers().nodes().stream().filter(n -> n.id() != grid0.localNode().id())
```

Comment:
grid0.cluster().forServers().forRandom().node();
I believe it will be more simple.